devnet/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+services: 
+  alice:
+    build:
+      context: ./..
+      dockerfile: devnet/alice.Dockerfile
+      args:
+        DD_API_KEY: ${DD_API_KEY}
+        pubip: 10.5.0.2
+    ports:
+      - "7001"
+      - "8545"
+      - "8546"
+      - "8540"
+      - "9876"
+    networks:
+      vpcbr:
+        ipv4_address: 10.5.0.2
+
+  bob:
+    build:
+      context: ./..
+      dockerfile: devnet/bob.Dockerfile
+      args:
+        DD_API_KEY: ${DD_API_KEY}
+        key: "bob"
+        pubip: 10.5.0.3
+    ports:
+      - "7001"
+      - "8545"
+      - "8546"
+      - "8540"
+      - "9876"
+    networks:
+      vpcbr:
+        ipv4_address: 10.5.0.3
+    depends_on:
+      - "alice"
```

Comment:
When do we know when Alice is ready? Maybe should we have a shell/Go entrypoint to wait for Alice to be ready for bob and charlie? Or will e.g. `bob` fail if `alice` is not up yet?

Reply:
In @noot PR for the `--babe-lead` flag, she changed the behaviour for all other non-lead auth nodes to wait for the first block.  So we still wait for alice to start up first, but it's not completely necessary.

Reply:
Awesome, let's leave those `depends_on` for documentation really then :wink: